proto/tendermint/crypto/proof.proto

```diff
@@ -27,7 +27,7 @@ message DominoOp {
 }
 
 // ProofOp defines an operation used for calculating Merkle root
-// The data could be arbitrary format, providing nessecary data
+// The data could be arbitrary format, providing necessary data
```

Comment:
As mentioned [here](https://github.com/cosmos/cosmos-sdk/pull/13419#pullrequestreview-1124874372), it is better to make a PR for that in the tendermint repo. Next time we do `make proto-update-deps` this will be overwritten.